src/utils.rs

```diff
@@ -45,3 +45,48 @@ pub fn load_expected_from_replay_filename(replay_filename: String) -> String {
     let expected_filename = replay_filename.replace(".replay", ".expected");
     std::fs::read_to_string(expected_filename).unwrap().trim().into()
 }
+
+
+
+
+/// Will create a data structure from the forex conversion pairs
+pub fn extract_conversion_pairs<'de, D>(deserializer: D) -> Result<std::collections::BTreeMap<String, f64>, D::Error>
```

Comment:
Sorry for the late reply. Will change the formatting of the function. 

I'm not 100% sure what you mean with the error handling. The de serialization error is returned if it fails. The other unwraps should not panic, since the process is checking weather they are and object or f64 or i64. Or am i missing something ?

Reply:
I think i missed that you were already checking `value.get("quote").is_some()` before calling `value.get("quote").unwrap()`, so this should be fine.